### DISCUSSION TITLE
r68S已经通过SSR Plus让家里的所有设备可以科学上网了，远程的电脑也能通过pptp vpn连接到家里的网络了，连接后不能科学上网，我应该怎么设置？谁来指导一下？

### DISCUSSION THREAD

**Discussion by zzxysun** [coolsnowwolf/lede#9969] [R68S / 电犀牛专区]
r68S已经通过SSR Plus让家里的所有设备可以科学上网了，远程的电脑能通过路由器的pptp也能连接家里的网络，但是不能科学上网。我应该怎么设置openwrt才可以让远程的电脑通过pptp连接家里的网络后，也能科学上网？谁会？来指导一下呗。之前，我用OpenClash科学上网后后，什么都不用设置，远程的电脑通过pptp连接家里的网络后，也能科学上网，但是上网时间控制服务就不起作用了。谁能指导指导我应改怎么设置呀？

**Comment by HongJie-Master** [comment]
你可以使用openvpn或wireguard，我试了这俩个均可以连到家里网络和科学上网

**Reply by shzstone** [reply]
访问控制里面设置，把VPN勾选上看行不行

**Reply by HahhCH** [reply]
我也遇到了，访问控制里勾选vpn那个接口，然后改成绕过大陆ip才能分流

**Reply by fredzhu-greendot** [reply]
远程电脑直接挂SSR上不行吗？为啥一定要回家里绕一圈

**Comment by GreatMichaelLee** [comment]
pptp没用，你用zerotier或者Openvpn吧，openvpn server我架在openwrt软路由上，我的是这个配置，自己悟吧，openvpn自己的网段 是10.0.2.0/24网段，家庭内网LAN是192.168.2.0/24网段，软路由内网LAN IP是192.168.2.1，dns也是软路由 LAN IP. 客户端关键三个点：一是吧lan网段的路由推送过去，二是bypass dhcp，三是把lan的dns推给客户端。

![image](https://github.com/coolsnowwolf/lede/assets/20067401/4201a6df-fdc2-485b-95ec-670b6d9b3978)
